lib/html_parser.dart

```diff
@@ -78,6 +78,11 @@ class HtmlParser extends StatelessWidget {
       textSpan: parsedTree,
       style: cleanedTree.style,
       textScaleFactor: MediaQuery.of(context).textScaleFactor,
+      renderContext: RenderContext(
+        buildContext: context,
+        parser: this,
+        style: Style.fromTextStyle(Theme.of(context).textTheme.bodyText2),
```

Comment:
Is this the style we use from everywhere else? Don't want text to get all strange with inconsistent styling

Reply:
This one honestly doesn't matter, the style in `RenderContext` doesn't get used at all when building the `StyledText`. I'll remove that line since it isn't really necessary.